### PR TITLE
Make a responsive design for tablet and mobile devices

### PR DESCRIPTION
**Pull request by AwaMelvine**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1,3 +1,355 @@
...
+.main-header .intro span {
+  display: block;
+}
+
+.main-header .intro .intro-text {
+  font-size: 5rem;
+  font-family: "Bangers", cursive;
+  text-align: center;
+  padding-left: 2rem;
+}
```
Comment (mattshardman) [inline css/index.css:215]: The padding left should be set to 0 in mobile as it is causing the text to be slightly out of line

File: css/services.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+/* Services Header */
+.services-header {
+  display: flex;
+  flex-direction: column;
+  border-bottom: 1px solid black;
+}
+
+.services-header .services-header-img {
+  flex: 1;
+}
```
Comment (mattshardman) [inline css/services.css:11]: I would set this to width: 100% instead of flex:1, this means the image keeps its correct proportions

**Comment by mattshardman** [comment]
Amazing job Melvine, this is very nearly perfect!

* Your design closely matches the design files.
* You've correctly implemented media queries.
* Your code is clean, well commented and easy to read.
* Your git workflow is superb.
* You have gone beyond the mvp and completed stretch goals.

**Possible things to improve**
* Watch how padding and margins effects layout in mobile, it may need to be removed to ensure things are correctly aligned.
* Make sure images retain their correct proportions when resized.

Overall absolutely awesome job, (I almost feel mean adding things to improve!). Keep it up!

**Comment by AwaMelvine** [comment]
Thanks, Matt. I appreciate all your feedback and will put it into effect going forward!

**Pull request merged**